test/e2e/http_deploy_test.go

```diff
@@ -0,0 +1,183 @@
+package e2e
+
+import (
+	"fmt"
+	cli "github.com/boson-project/func/test/cli"
+	"github.com/boson-project/func/test/e2e/utils"
+	"os"
+	"strings"
+	"testing"
+	"time"
+)
+
+const (
+	registry = "localhost:5000/func"
+	// namespace = "func" // using current namespace set for integration (func)
```

Comment:
Why is this commented?

Reply:
It was not in use. It is temporarily commented due lint issues. I'm still working in a better way to handle and deal with namespaces on e2e tests.